Containerfile

```diff
@@ -1,12 +1,13 @@
-# Single stage: Ubuntu 24.04 with Python 3.12 and Node.js 22
+# Single stage: Ubuntu 24.04 with Python 3.13 and Node.js 22
 FROM ubuntu:24.04
 
 WORKDIR /app
 
-# Install Python 3.12
-RUN apt-get update && apt-get install -y \
-    python3.12 \
-    python3.12-venv \
+# Install Python 3.13
+RUN add-apt-repository ppa:deadsnakes/ppa \
```

Comment:
Looks like this new line is failing CI - was this needed to get access to a Python 3.13 apt repo or something?

Reply:
Alright from some quick googling I see the answer is yes - weird that there's no official Python 3.13 repo for Ubuntu - regardless, adding something like this may be the fix 
```
sudo apt install software-properties-common
```